ext/src/main/java/com/playone/mobile/ext/InvlideValue.kt

```diff
@@ -1,4 +1,6 @@
 package com.playone.mobile.ext
 
 const val defaultInt = -1
+const val defaultLong = -1L
+const val defaultDouble = -1.0
 const val defaultStr = "invalidString"
```

Comment:
What do you think, use UPPER_SNAKE_CASE for constant variable? 
Ref:
https://android.github.io/kotlin-guides/style.html#constant-names